src/main/java/pl/futurecollars/invoice/db/DatabaseConfiguration.java

```diff
@@ -0,0 +1,67 @@
+package pl.futurecollars.invoice.db;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import pl.futurecollars.invoice.db.file.FileRepository;
+import pl.futurecollars.invoice.db.memory.MemoryRepository;
+import pl.futurecollars.invoice.service.InvoiceService;
+import pl.futurecollars.invoice.utils.FilesService;
+import pl.futurecollars.invoice.utils.IdService;
+import pl.futurecollars.invoice.utils.JsonService;
+
+@Configuration
+public class DatabaseConfiguration {
+
+  @Bean
```

Comment:
fileService oraz jsonService są potrzebne tylko w przypadku plikowej bazy danych

Reply:
Dałem teraz warunek dla tych 2 bean'ów tylko dla plikowej bazy danych

Reply:
Z tego co widzę w kodzie to warunków brak, za to pojawiły się adnotacje @Service w obu tych klasach.